components/image/image.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../shared/reset.css">
+    <link rel="stylesheet" href="../../shared/navbar.css" />
+    <link rel="stylesheet" href="../../shared/sidebar.css">
+    <link rel="stylesheet" href="../../shared/root.css" />
+    <link rel="stylesheet" href="../../shared/components.css">
```

Comment:
Suggestion: rather than importing so many files every time to every component, you can import all into one CSS file in sequence and then import only that CSS file into the component.

Reply:
Thanks for the suggestion, will implement it in the next fix!